static/less/misc.less

```diff
@@ -5,4 +5,3 @@
   text-overflow: ellipsis;
   white-space: nowrap;
 }
-
```

Comment:
Hmm, not sure why I got these changes. I just merged in master and the precommit hooks suggested this change.